allensdk/brain_observatory/behavior/swdb/utilities.py

```diff
@@ -355,11 +382,41 @@ def get_active_cell_indices(dff_traces):
 
 
 def compute_lifetime_sparseness(image_responses):
-    # image responses should be an array of the trial averaged responses to each image
-    # sparseness = 1-(sum of trial averaged responses to images / N)squared / (sum of (squared mean responses / n)) / (1-(1/N))
+    # image responses should be an array of the trial averaged responses
+    # to each image
+    # sparseness = 1-(sum of trial averaged responses to images / N)squared /
+    # (sum of (squared mean responses / n)) / (1-(1/N))
     # N = number of images
     # after Vinje & Gallant, 2000; Froudarakis et al., 2014
     N = float(len(image_responses))
-    ls = ((1 - (1 / N) * ((np.power(image_responses.sum(axis=0), 2)) / (np.power(image_responses, 2).sum(axis=0)))) / (
+    ls = ((1 - (1 / N) * ((np.power(image_responses.sum(axis=0), 2)) /
+     (np.power(image_responses, 2).sum(axis=0)))) / (
         1 - (1 / N)))
     return ls
+
+def literal_col_eval(df: pd.DataFrame,
```

Comment:
Here are the two utilities functions added

Reply:
swdb is summer workshop of the dynamic brain. It doesn't belong here :)

Reply:
Ah, I thought SWDB might've meant software db. (:
How about creating a utilities.py in `allensdk/core` ?

Reply:
Sure :)